compiler-core/src/error.rs

```diff
@@ -223,6 +223,13 @@ pub enum Error {
         package: String,
         build_tools: Vec<String>,
     },
+
+    #[error("The package {package} requires gleam version {required_version} and you are using {gleam_version}")]
```

Comment:
```suggestion
    #[error("The package {package} requires Gleam v{required_version} and you are using v{gleam_version}")]
```

Reply:
Required version is going to be a range, so this interpolates as `Gleam v~>1.0`.

Reply:
Required version is going to be a range, so this interpolates as `Gleam v~>1.0`.

Reply:
Oh! Thank you